pandas/core/frame.py

```diff
@@ -894,6 +893,10 @@ def itertuples(self, index=True, name="Pandas"):
             The name of the returned namedtuples or None to return regular
             tuples.
 
+        Returns
+        -------
+        Returns namedtuples.
```

Comment:
In this case instead of `Returns`, can you use `Yields`, as the function is a generator, and the namedtuples are yielded from it.

The exact syntax would be something like:
```
Yields
------
collections.namedtuple
    A description of what is being yielded.
```

Reply:
Got it! Working on it now.